src/modules/common/containers/LanguageNotAvailableContainer.js

```diff
@@ -1,51 +1,11 @@
 // @flow
 
-import { connect } from 'react-redux'
-import type { Dispatch } from 'redux'
-import type { StateType } from '../../../modules/app/StateType'
-import type { StoreActionType, SwitchContentLanguageActionType } from '../../../modules/app/StoreActionType'
-import LanguageNotAvailablePage from '../../../modules/common/components/LanguageNotAvailablePage'
+import LanguageNotAvailablePage from '../components/LanguageNotAvailablePage'
 import withTheme from '../../theme/hocs/withTheme'
 import compose from 'lodash/fp/compose'
 import { translate } from 'react-i18next'
-import createNavigateToCategory from '../../app/createNavigateToCategory'
-
-const mapStateToProps = (state: StateType) => {
-  const languages = state.cityContent.languages
-  if (!languages) {
-    throw new Error('languages have not been set.')
-  }
-
-  return {
-    city: state.cityContent.city,
-    languages
-  }
-}
-
-const mapDispatchToProps = (dispatch: Dispatch<StoreActionType>, ownProps) => {
-  return {
-    changeLanguage: (city: string, newLanguage: string) => {
-      const switchContentLanguage: SwitchContentLanguageActionType = {
-        type: 'SWITCH_CONTENT_LANGUAGE',
-        params: {
-          city,
-          newLanguage
-        }
-      }
-      dispatch(switchContentLanguage)
-      createNavigateToCategory('Dashboard', dispatch, ownProps.navigation)({
```

Comment:
This prop is not passed to the now undefined because only `changeLanguage` is passed to `LanguageNotAvailableContainer` in the hoc.

If I'm right then you can try to fix the flow types here.

Reply:
`createNavigateToCategory` isn't a prop here, but part of the code that is executed on `changeLanguage`, so it should be fine.